src/components/elements/grid/types.ts

```diff
@@ -0,0 +1,9 @@
+import { ContainerProps } from "@lifesg/react-design-system/layout";
+import { TComponentOmitProps, TFrontendEngineFieldSchema } from "../../frontend-engine";
+import { IBaseFieldSchema } from "../../fields/types";
+
+export interface IGridSchema<V = undefined>
+	extends Omit<IBaseFieldSchema<"grid", V>, "label">,
+		TComponentOmitProps<ContainerProps, "children" | "type"> {
+	children: Record<string, TFrontendEngineFieldSchema<V>>;
+}
```

Comment:
- `IBaseFieldSchema` is meant for fields but grid is an element, hence it should extend from `IBaseElementSchema` instead
- omit `stretch` from `ContainerProps` because we are overriding its padding